packages/components/action-link/_action-link.scss

```diff
@@ -1,3 +1,11 @@
+@use "../../core/settings/colours" as *;
+@use "../../core/settings/typography" as *;
+@use "../../core/tools/focused" as *;
+@use "../../core/tools/mixins" as *;
+@use "../../core/tools/sass-mq" as *;
+@use "../../core/tools/spacing" as *;
+@use "../../core/tools/typography" as *;
```

Comment:
These lines are largely repeated for every component. Could we simplify by allowing all the tools to be imported in a single line?

Maybe something like

```
@use "../../core/tools/all" as *;
```

Reply:
Yep, being able to do `@use "../../core/tools" as *;` sounds good to me.